src/css/checkbox.css

```
@@ -0,0 +1,72 @@
+.checkbox input[type='checkbox'] {
+  opacity: 0;
+}
+
+.checkbox--disabled {
+  cursor: not-allowed;
+  opacity: 0.2;
+}
+
+.checkbox label {
+  position: relative;
+  display: inline-block;
+  padding-left: 24px;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+}
+
+.checkbox label::before,
+.checkbox label::after {
+  position: absolute;
+  display: inline-block;
+  content: '';
+}
+
+.checkbox label::before {
+  top: 4px;
+  left: 0;
+  width: 16px;
+  height: 16px;
+  border: 2px solid var(--color-moon-500);
+  border-radius: 2px;
+}
+
+.checkbox label::after {
+  top: 8px;
+  left: 3px;
+  width: 11px;
+  height: 6px;
+  transform: rotate(-45deg);
+  color: white;
```

We normally use variables even for white values...

```suggestion
  color: var(--color-space-100);
```

---

Done on commit https://github.com/magnetis/astro/pull/27/commits/3d3459e987c8d4aa3a3e501872c1d3acf1953d4a .